### PR TITLE
Create retire-unpublished.md

### PR DESCRIPTION
**Pull request by hot007**
I have not added this file to the toc until it's been reviewed!!! Please add on merge if approved @paolap or @chloemackallah 

I am not happy with the number rendering here where I've done 'custom' things, if you wish to rework the numbered lists I think it'd help?

This commit attempt to address most of Sharon's issues raised in #10

### CODE REVIEW

File: Governance/retire/retire-unpublished.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Intermediate data products
+
+How each of these scenarios is handled will typically be determined on a project basis, with a view to the importance of **reproducibility** and considering relative **compute or storage costs**.
+
+## Suggested procedures
+
+**If compute is readily available but storage is limited**
+
+1. Maintain a database or wiki of model runs
+2. Create zip archives of model configurationsand move to slow access tape storage if they are required to be kept for reproducibility
```
Comment (chloemackallah) [inline Governance/retire/retire-unpublished.md:18]: "configurations and"

Comment (hot007) [reply Governance/retire/retire-unpublished.md]: Fixed, thank you.

**Review by paolap — CHANGES_REQUESTED** [review]
I only fixed one misspelling. 
My main concern with this is that is looking at a very specific use case
"plethora of data that is associated with published data creation."
and it is looking at this mostly from a model run point of view.
I think it would benefit from being more generalised. We do have a current issue with lots of data, both model and analysis generated, that is re-use by the community. So basically unpublished data which is not versioned but has been used in publications. 
In conclusion, I'm happy for this to be merged as it is, but then we should discuss at the meeting if we want to generalise this example or add another "unpublished" use case to cover what I described above. In first instance it might even encourage publication.

Another action we should add regardless is "documenting the process" if this data isn't published how we document its provenance. It could be included in associated published data documentation, or you might need a metadata record somewhere which refers specifically to this a separate dataset.

**Comment by hot007** [comment]
I very much agree - I think it'd be great to generalise or if it's substantially different, add a whole extra example. 
The second use case you describe certainly might be used as a lever to encourage publication for the purposes of reuse!

And yes, we haven't actually captured provenance here, and i wonder if it belongs more in the "creating" section than "publishing" or "retiring", but it maybe should be mentioned throughout??

**Comment by paolap** [comment]
The provenance I'm thinking of is about documenting the "retirement" step. It is something that should be added to any use case to complete the dataset information, in a way that if someone asks about the data it's easy to query it and let them know what happen to the data and why, and if there's an alternative way to retrieve it, i.e. from deep storage

**Pull request merged**